notebooks/ai-ml/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Nit: "load the data into Spark" -&gt; "load file paths into Spark"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>